snappy/binaries.go

```diff
@@ -90,7 +90,7 @@ ubuntu-core-launcher {{.UdevAppName}} {{.AaProfile}} {{.Target}} "$@"
 	var templateOut bytes.Buffer
 	t := template.Must(template.New("wrapper").Parse(wrapperTemplate))
 	wrapperData := struct {
-		AppName     string
+		SnapName    string
 		AppArch     string
 		AppPath     string
```

Comment:
All of those will be addressed separately to make the review short and quick each time.